packages/forms/src/Form.js

```diff
@@ -191,4 +191,6 @@ Form.defaultProps = {
 	buttonBlockClass: 'form-actions',
 };
 
+Form.displayName = 'TalendForm';
```

Comment:
that default value happens only in dev. in prod the function name is minified.
And in snapshot without displayName I have <unknown />

https://github.com/Talend/data-streams/commit/d2554159919335e29312bf7b2b5e765e7c15f117#diff-8ac575987465edbb6a6b7810963bc2c8R24